sdk/src/aurora/contract.rs

```diff
@@ -106,6 +107,40 @@ impl ContractMethod for SetSiloParams {
     }
 }
 
+pub struct SetFixedGas {
+    pub args: FixedGasArgs,
+}
+
+impl ContractMethod for SetFixedGas {
+    type Response = ();
+
+    fn method_name(&self) -> &'static str {
+        "set_fixed_gas"
+    }
+
+    fn params(&self) -> Result<Vec<u8>, std::io::Error> {
+        borsh::to_vec(&self.args)
+    }
+}
+
+// temporarly until engine 4.0.0 release
+#[near(serializers = [borsh])]
```

Comment:
```suggestion
#[derive(Debug, BorshDeserialize, BorshSerialize)]
```
And you don't need to add the `near-sdk-rs` dep.